### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by fsepteixeira** [fsept11/FoolHD#4]
Hello, thanks for your code! Would you like to tell me what the loss function you use for training the Audionet, it will be great helpful for me~

_Originally posted by @Ian-Tam in https://github.com/fsepteixeira/FoolHD/issues/2#issuecomment-1004613325_

Hi @Ian-Tam,

I've created a new issue so that we don't mix up questions. Can I ask you what you mean by the Audionet?

**Comment by Ian-Tam** [comment]
Ohhhh, sorry, I mixed up the model between your code and the other one. I find that you upload a py file, maybe I can get my answer after reading it~ thanks a lot.
